### DISCUSSION TITLE
pip install -r requirements.txt isn't working

### DISCUSSION THREAD

**Discussion by PatLeeNic** [UltimaHoarder/UltimaScraper#824] [Q&A]
I've been trying all day to get this to work on MacOS. 

I keep getting `command not found: pip` for `pip install -r requirements.txt` and I don't know why. I've installed visual studio but I don't think it included the C++ build stuff and I don't know how to get that on MacOS. 

I tried just `python start_ofd.py` but then got this after:

```
Traceback (most recent call last):
  File "start_ofd.py", line 2, in <module>
    import tests.main_test as main_test
  File "/Users/patnichols/OnlyFans-6.3.2 2/tests/main_test.py", line 10
    python_version = f"{version_info.major}.{version_info.minor}"
                                                                ^
SyntaxError: invalid syntax
```

I really really don't understand. I'm really really bad at this shit. I got a D for computer science like 5 years ago and I've barely coded since. I am losing patience rapidly. This is like the third or fourth different thing I've tried for archiving my onlyfans subs and I'm losing hope. Oh and .profiles doesn't appear in the project even after I've unhidden the files. So I don't know where the hell that is.

**Comment by RonnieBlaze** [comment]
pip is a python command not visual stupid. did you install python for mac?

**Reply by PatLeeNic** [reply]
yeeeeep. Also the readme told me to install visual too didn't it?

**Reply by foomaster-oss** [reply]
@PatLeeNic @RonnieBlaze 

Any fix on this? I'm getting a  
`ERROR: Could not find a version that satisfies the requirement win32-setctime (from versions: none)
ERROR: No matching distribution found for win32-setctime
`
When I run pip install -r requirements.txt ... looks like win32-setctime is a windows only module. I'm there are people on mac OS who got this running
Running python 3.9

**Comment by headphonia** [comment]
Not sure which version of macOS you're using, but Mojave only has python 2 installed, so you need to install python 3. Read and follow this guide, should help you get python 3 installed.

https://opensource.com/article/19/5/python-3-default-mac#what-to-do
